pages/login.php

```diff
@@ -158,13 +163,21 @@
       </div>
       <p>Don't have an account? <a href="signup.php">Sign Up</a></p>
 
-    
 
 
-			
+
+
 	</form>
   </div>
   <script src="../js/showpassword.js"></script>
+  <script>
+  $(".toggle-password").click(function() {
+   $(this).toggleClass("fa-eye fa-eye-slash");
+   if ($("#pswd").attr("type") == "password") {
+    $("#pswd").attr("type", "text");
+   } else {
+     $("#pswd").attr("type", "password");
+   }
+  });
+  </script>
```

Comment:
The lines `175 - 180` are repeated code. You can create a function as shown below and call it three times.
```js
function showPassword(id) {
   $(this).toggleClass("fa-eye fa-eye-slash");
   if ($(id).attr("type") == "password") {
    $(id).attr("type", "text");
   } else {
     $(id).attr("type", "password");
   }
}
```

This can be called three times with `#pswd,` `#pswd2` as arguments each time. Also, create a new file named `showPassword.js` inside the `js` folder of the repo and include this here.

Reply:
If you have any doubts or If you didn't understand what I told you. Feel free to ask me on the discord server. I will be more active over there.

Reply:
ok